shared-libs/search/src/generate-search-requests.js

```diff
@@ -1,18 +1,14 @@
 const _ = require('lodash/core');
-_.partial = require('lodash/partial');
-_.partial.placeholder = _;
 const moment = require('moment');
+
 const END_OF_ALPHABET = '\ufff0';
+const MINIMUM_SEARCH_TERM_LENGTH = 3;
 
-const getKeysArray = function(keys) {
-  return keys.map(function(t) {
-    return [ t ];
-  });
-};
+const getKeysArray = (keys) => keys.map(t => [ t ]);
```

Comment:
```suggestion
const getKeysArray = (keys) => keys.map(term => [ term ]);
```
I wonder why Sonar didn't catch single-letters variable names. Maybe the [rule is off](https://rules.sonarsource.com/javascript/RSPEC-117/?search=name)?

Reply:
Replaced with `key` instead of `term`.